ml4h/plots.py

```diff
@@ -2733,6 +2737,93 @@ def plot_precision_recalls(predictions, truth, labels, title, prefix="./figures/
     plt.savefig(figure_path)
     logging.info("Saved Precision Recall curve at: {}".format(figure_path))
 
+def plot_dice(predictions, truth, labels, paths, title, prefix="./figures/", dpi=300, width=3, height=3):
+    label_names = labels.keys()
```

Comment:
Add Docstring and ideally type hints, though we have not done a great job of that in general.